### PR TITLE
libxml++4 4.0.0 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libxml++4.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+      int main(int argc, char *argv[])
+      {
+         xmlpp::Document document;
+         document.set_internal_subset("homebrew", "", "https://www.brew.sh/xml/test.dtd");
+         xmlpp::Element *rootnode = document.create_root_node("homebrew");
+         return 0;
+      }
+    EOS
+    ENV.libxml2
```
Comment (SMillerDev) [inline Formula/libxml++4.rb:40]: I don't think this exists

Comment (tschoonj) [reply Formula/libxml++4.rb:40]: That's fine, this example is just constructing a simple XML tree, it is not actually trying to fetch the url.

Comment (SMillerDev) [reply Formula/libxml++4.rb:40]: ? I mean the ENV statement, I don't think it does anything

Comment (tschoonj) [reply Formula/libxml++4.rb]: Aha, I removed that line, seems unnecessary on my local machine.

**Comment by SMillerDev** [comment]
Same issue as the version 5, it should be a versioned formula.

**Comment by tschoonj** [comment]
Closed in favour of #68071

**Pull request closed without merging**